README.md

```diff
@@ -1,8 +1,131 @@
+
+**Familiar:** C, Prolog for AI, PHP/React/Node JS, R / Hadoop, Axure / GCP, SAS Enterprise Guide / Cognos / MicroStrategy BI, VBA
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_JjFalcon](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/linux_sql)]: 
+ Developed a Linux Cluster Monitoring Agent using bash and psql CLI in CentOS to collect the hardware specifications and monitor the resource usage of nodes connected to a Linux Cluster. 
+ Deployed the application using Docker which is used to create a PostgreSQL instance to manage both static and dynamic data collection stored in RDBMS that can later be queried to produce reports for resource planning.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/core_java)]:
+      
+  - Twitter App: Developed an app that utilizes the Twitter and RESTful APIs using the MVC architecture to enable a Twitter user to fetch, post, and delete tweets.  Tested using Maven's JUnit testing.
+  - JDBC App: Developed an application that demonstrates the DAO/DTO implementation of JDBC to a PostgreSQL RDBMS.  Implemented and tested basic CRUD and DML operations.
+  - Grep App: Developed an application that simulates the Unix egrep -R implementation that searches all files recursively from a given directory and outputs a file containing all the lines that match the given regex pattern.  Deployed via Docker.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/springboot)]: COMING SOON
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/python_data_anlytics)]: COMING SOON
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/hadoop)]: COMING SOON
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/spark)]: COMING SOON
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JjFalcon/tree/master/cloud_devops)]: COMING SOON
+
+
+## Highlighted Projects
+**Coop Work Projects: AODA Converter** [[GitHub](https://github.com/f57c0n/JB-aodaExcel)]: Developed an application program that takes in an Excel file as input.  The program reads and convert the file to be AODA [Accessibility for Ontarians with Disabilities Act] compliant and saves it to a new Excel file. 
+
+
+**Coop Work Projects:  SQL Loader** [[GitHub](https://github.com/f57c0n/JB-sqlLoader)]: Developed an application program that takes in a CSV file as input.  The program reads and analyze the file to detect column data types and remove invalid entries, and provide DML options to review the file and a commit option to create and insert data into a remote database.  It also outputs a file with rows containing invalid entries.
```

Comment:
Developed a python/java/language application program (what language?) that reads and analyzes the file blah blah...

I need to see more tech terms rather than describing how program works / features.